packages/query/src/configure/configure.module.ts

```diff
@@ -85,6 +86,17 @@ export class ConfigureModule {
         pgClient._explainResults = [];
       });
     }
+    // todo: support ssl
+    const dataSource = new DataSource({
```

Comment:
Can we specify this as read only?

Reply:
I couldn't find a way. I think user may have to set up the readonly permission on db user